.github/workflows/tests.yml

```diff
@@ -12,73 +12,57 @@ env:
     -Porg.gradle.parallel=false
 
 jobs:
-  # Build Android Tests
-  build-android-tests: 
-    name: Build Android Tests 
```

Comment:
Can we just disable this job, in case we need to easily enable it again for some reason?

Reply:
I can leave it in but commented out; but it will be in github history anyway ?

Reply:
OK, thanks. Yes, still visible in the git history, but a bit more hidden, so probably quite lost.
I was wondering if something like this exists: `if: never()` to disable a job.